flocker/node/agents/ebs.py

```diff
@@ -300,6 +281,7 @@ def _wait_for_new_device(base, size, time_limit=60):
                 return new_device
         time.sleep(0.1)
         elapsed_time = time.time() - start_time
+    NO_NEW_DEVICE_IN_OS(devices=base, size=size, time_limit=time_limit).write()
```

Comment:
If a new device with the expected size does not appear within 60s then it'd be useful to log the actual list of devices and their sizes, I think. ie a new device has shown up, but with a slightly unexpected size.